src/java/org/apache/cassandra/service/opstate/KeyspaceTableOpStatePersister.java

```diff
@@ -34,127 +30,94 @@
 import org.slf4j.LoggerFactory;
 
 import com.fasterxml.jackson.core.JsonParseException;
-import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.palantir.cassandra.utils.FileParser;
 
 public class KeyspaceTableOpStatePersister
 {
     private static final Logger log = LoggerFactory.getLogger(KeyspaceTableOpStatePersister.class);
 
-    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
-
-    private File persistentFile;
     private final Path persistentFileLocation;
 
+    private final FileParser<Map<String, Long>> parser;
+
     public KeyspaceTableOpStatePersister(Path persistentFileLocation)
     {
         this.persistentFileLocation = persistentFileLocation;
-        this.persistentFile = getOrMaybeCreateStateFile();
+        this.parser = new FileParser<>(persistentFileLocation, new TypeReference<Map<String, Long>>()
+        {
+        });
```

Comment:
Nit: inline bracket + parentheses

Reply:
This comes from the default formatter when I set up the repo. Is there a specific formatter that I should use?

Reply:
Ugh ... No the formatter is correct, this is following Cassandra style, it's just ugly imo